docs/metrics/getting-started-aspnetcore/Program.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+
+var appBuilder = WebApplication.CreateBuilder(args);
+
+// Configure OpenTelemetry with metrics and auto-start.
+appBuilder.Services.AddOpenTelemetry()
+    .ConfigureResource(builder => builder
+        .AddService(serviceName: "OTel.NET Getting Started"))
+    .WithMetrics(builder => builder
+        .AddAspNetCoreInstrumentation()
+        .AddConsoleExporter((exporterOptions, metricReaderOptions) =>
+        {
+            metricReaderOptions.PeriodicExportingMetricReaderOptions.ExportIntervalMilliseconds = 1000;
+        })
+        .AddPrometheusExporter());
+
+var app = appBuilder.Build();
+
+app.UseOpenTelemetryPrometheusScrapingEndpoint(context => context.Request.Path == "/internal/metrics");
```

Comment:
I think there are two reasons that lead me to the current approach:
1. The existing getting-started tutorial for Prometheus/Grafana doesn't show the best practice for ASP.NET Core app.
2. We suppress instrumentation for Prometheus Exporter, I'm curious if we could avoid the suppression for metrics.

Reply:
1. Lets modify it to use asp.net core app with prometheus exporter (Not the HttpListener one)
2. This can be separately discussed. Lets not use a getting-started guide to be more complex than needed.

Reply:
Agree, I've removed the Prometheus part from this PR.